### ISSUE TITLE
[Bug] the attention mask should be reversed

### ISSUE THREAD

**Issue by Weiyun1025** [lucidrains/halonet-pytorch#4]
`Tensor.masked_fill_(mask, value)` fills elements of self tensor with value where mask is True. So your code in line 145 should be `mask = ~mask.bool()`.

**Comment by 000wahab000** [comment]
can i try this issue?
